packages/core/src/anchor/ethereum/__tests__/steph-temp.test.ts

```diff
@@ -0,0 +1,179 @@
+import { jest, test, expect } from '@jest/globals'
```

Comment:
can we either delete this file or clean it up and rename it?

Reply:
deleted